src/ValidationException.php

```diff
@@ -22,32 +22,22 @@ public function __construct($errors, $model = null, $intent = null)
     {
```

Comment:
Please set array type for $errors argument of constructor

Reply:
done